File: output/anthropic_claude_sonnet_4_6/few/cpp/Cobra coral.cpp
#include <bits/stdc++.h>
using namespace std;
int main(){
    int a,b,c,d;
    cin>>a>>b>>c>>d;
    // Verdadeira: BVBPBVBP -> pattern positions 0,1,2,3 = B,V,B,P
    // So a==c and b!=d and a!=b and a!=d and b!=c
    // Falsa: BVPBVPBV -> pattern positions 0,1,2,3 = B,V,P,B
    // So a==d and a!=b and a!=c and b!=c and b!=d and c!=d
    // Verdadeira pattern: x y x z where x!=y, x!=z, y!=z
    // Falsa pattern: x y z x where x!=y, x!=z, y!=z
    if(a==c && a!=b && a!=d && b!=d){
        cout<<"V"<<endl;
    } else {
        cout<<"F"<<endl;
    }
    return 0;
}

File: output/csbc_semish_2026/anthropic_claude_sonnet_4_6/few/python/Cobra coral.py
import sys
a, b, c, d = map(int, sys.stdin.read().split())

# Verdadeira: BVBPBVBP -> padrão: pos0==pos2, pos1!=pos0, pos3!=pos1, pos3!=pos2, pos1!=pos3
# Falsa: BVPBVPBV -> padrão: pos0==pos3, pos1!=pos0, pos2!=pos1, pos2!=pos0

# Verdadeira coral pattern: a b a c (where a!=b, a!=c, b!=c)
# a==c, a!=b, a!=d, b!=d
if a == c and a != b and a != d and b != d:
    print("V")
else:
    print("F")